src/sentry/integrations/jira/webhooks.py

```diff
@@ -16,6 +21,102 @@ class JiraIssueUpdatedWebhook(Endpoint):
     def dispatch(self, request, *args, **kwargs):
         return super(JiraIssueUpdatedWebhook, self).dispatch(request, *args, **kwargs)
 
+    def handle_assignee_change(self, integration, data):
+        assignee = data['issue']['fields']['assignee']
+        issue_key = data['issue']['key']
+
+        if assignee is None:
+            sync_group_assignee_inbound(
+                integration, None, issue_key, assign=False,
+            )
+        else:
+            sync_group_assignee_inbound(
+                integration, assignee['emailAddress'], issue_key, assign=True,
+            )
+
+    def handle_status_change(self, integration, data):
```

Comment:
Should this functionality be shared across integrations similar to `sync_group_assignee_inbound`?